radio/src/targets/horus/hal.h

```diff
@@ -1057,6 +1057,9 @@
   #define BT_USART_GPIO                 GPIOG
   #define BT_TX_GPIO                    GPIO_PIN(GPIOG, 14) // PG.14
   #define BT_RX_GPIO                    GPIO_PIN(GPIOG, 9)  // PG.09
+#if defined(RADIO_TX16S)
+  #define BT_PWR_GPIO                   GPIO_PIN(GPIOB, 0) // PB.00
+#endif
```

Comment:
Is this not also needed for T16?

Reply:
Unsure, do they have external aux port ?

Reply:
No, but reading through the CMake defines, they presently seem to be the only other horus radio to have AUX2 enabled except when when bluetooth, and *do* have facility for a bluetooth module to be added... 
![image](https://github.com/user-attachments/assets/4cd0c280-29b1-4443-9adc-f7c440881ed5)

Reply:
Ok, that's a different thing, tx16s does have similar pad, and they work without this PR, as they are supplied with permanent +5v. This PR is for cases where the 'external' aux2 port is used instead of internal pads

Reply:
Ok, thanks... Well, just need that other French dude to say he's happy with this and then can include in 2.10.3 also :)